Tests/CommanderTests/ArgumentDescriptionSpec.swift

```diff
@@ -18,7 +18,7 @@ public func testArgumentDescription() {
       let help = Help([
         BoxedArgumentDescriptor(value: Option<String>("opt1", "example")),
         BoxedArgumentDescriptor(value: Flag("flag1", description: "an example")),
-        BoxedArgumentDescriptor(value: Flag("flag2", default: true)),
```

Comment:
I think I prefer the label being explicit here. It's clearer to the reader what the value `true` is actually for.

Perhaps it would make more sense to make the `default` label explicit on all options. The reason it wasn't initially there for `Option` is because option MUST have a default value.